Sources/Purchasing/StoreKitAbstractions/SK2StoreProduct.swift

```diff
@@ -100,6 +85,33 @@ internal struct SK2StoreProduct: StoreProductType {
 
 }
 
+@available(iOS 15.0, tvOS 15.0, watchOS 8.0, macOS 12.0, *)
+private extension SK2StoreProduct {
+
+    var _currencyCode: String {
+        if #available(iOS 16.0, tvOS 16.0, watchOS 9.0, macOS 13.0, *) {
+            // This is marked as `@_backDeploy`, but it's not actually working before iOS 16.0
```

Comment:
lol why am I not surprised

Reply:
do we have a radar for this?

Reply:
Actually I figured out the reason: this only works in whatever version of the SDK added this, but you can't use if it you're compiling with Xcode 13.x obviously... but the docs don't even say when it was added.